pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -662,6 +663,29 @@ private RebalanceSummaryResult calculateDryRunSummary(Map<String, Map<String, St
     Set<String> serversRemoved = new HashSet<>();
     Set<String> serversUnchanged = new HashSet<>();
     Set<String> serversGettingNewSegments = new HashSet<>();
+    Map<String, RebalanceSummaryResult.TagInfo> tagsInfoMap = new HashMap<>();
+    String serverTenantName = tableConfig.getTenantConfig().getServer();
```

Comment:
Within `TenantConfig`, there is also `TagOverrideConfig` which you also want to include